### ISSUE TITLE
Passcode to lock the clipboard 

### ISSUE THREAD

**Issue by joecks** [joecks/clipboard-actions-community#36]
Protect the clips by adding a passcode which might be opened and closed when ever needed or time based, get inspired by Clip Stack

![0](https://cloud.githubusercontent.com/assets/180698/14137951/c131c3d8-f66a-11e5-8e06-4bde97348e57.png)

**Comment by prasaddaki** [comment]
Yes this is actually needed to maintain privacy and security about our copied clips....

**Comment by Viciouskitty** [comment]
I frequently use the Clipboard Actions app to copy and paste from my password manager app.  Yes, I frequently delete entries already used.  But a passcode for the clipboard would help secure my data.  Love this inspiration.

**Comment by 14URaPPs** [comment]
I forgot my 4 digit code how do I get in my clipboard
